tethys_cli/gen_commands.py

```diff
@@ -333,7 +333,7 @@ def write_path_to_console(file_path):
 }
 
 
-def generate_command(args):
+def generate_command(args, unknow_args=""):
```

Comment:
What happens when you provide an unknown argument now? Seems like it wouldn't catch it in the commands that don't use it. Perhaps good default behavior would be to combine these where `unknown_args` isn't used:

```
def generate_command(args, unknown_args=""):
    args = args + unknown_args
    ...
```

Reply:
Also, "unknow_args" -> "unknown_args"

Reply:
done